spectrum/templates/base.html

```diff
@@ -16,7 +16,11 @@
     <link rel="stylesheet" href="{% static 'info/css/global.css' %}">
     <link rel="stylesheet" href="{% static 'info/css/events.css' %}">
     <script src="https://kit.fontawesome.com/0b4fbfd8b7.js" crossorigin="anonymous"></script>
-    {% block stylesheets %}{% endblock %}
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.7.1/jquery.min.js"></script>           
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/js/bootstrap.bundle.min.js"></script>
```

Comment:
This is insecure without the integrity hash

Reply:
How to correct it. please let me know.

Reply:
There's an example later in the file that shows what it was previously, including an integrity hash.